transcom/mymove
# B-18115 updating prime sim sit extension

## [Agility ticket](tbd)

## Summary

Updating the Prime Simulator to have the functionality to send SIT extension requests.
Currently, there is an API already implemented in the Prime API, but the Prime Simulator is missing UI to send requests.

### How to test

1. Create a HHG shipment move, complete counseling, and approve as a TOO.
2. Change role to prime and go to the move you created in the Prime Simulator.
3. Create a new SIT service item for the shipment.
4. Approve SIT request as a TOO.
5. Go back to the Prime Simulator and a "Request SIT Extension" button should be visible in shipment details (only visible if there exists a SIT service item & is a HHG shipment.
6.  Click on the button and fill out the form in the Request SIT Extension page, then submit.

## Screenshots

![image](https://github.com/transcom/mymove/assets/146856854/b7a8d597-488a-4bf1-b495-dab1df2cdb70)
![image](https://github.com/transcom/mymove/assets/146856854/335ea636-4599-48ec-ac34-d31341051316)
![image](https://github.com/transcom/mymove/assets/146856854/345a8ca2-a393-41fa-bdc5-6e4c3c125c7d)
